src/modules/Chat/components/ChatMessagesBlock/index.ts

```diff
@@ -0,0 +1,37 @@
+import Block from '../../../../utils/Block';
+import template from './chatMessagesBlock.pug';
+import * as chatMessagesBlockStyles from './chatMessagesBlock.module.scss';
+import { ChatMessage, ChatMessageProps } from '../ChatMessage';
+
+export interface ChatMessagesBlockProps {
+    userId: string,
+    date: string,
```

Comment:
Скорее всего тут тип string, потому что надо как-то выводить дату на экран. Однако в логике все равно лучше оперировать типом Date, потому что он представляет распаршеную дату и огромное количество методов для работы с ней, нам не придется самим парсить строку, потому что это дико сложно и неудобно.
Я бы порекомендовал написать класс или функцию, например dateFormatter или getDisplayedDate, которая принимала бы объект Date и возвращала бы строку с датой в нужном формате. Еще вариант воспользоваться momentjs, классная библиотека (но я тебе этого не говорил, лучше реализовывать ручками 👽 )
```suggestion
    date: Date,
```